mail.py

```diff
@@ -21,24 +22,23 @@ def check_gmail():
         current_mail_id = int(file_reader.read())
 
         if possible_new_email_id > current_mail_id:
-            latest_email_id = possible_new_email_id
-            number_of_new_mails = latest_email_id - current_mail_id
-
             file_reader.close()
-
+            
+            latest_email_id = possible_new_email_id
```

Comment:
How can `possible_new_email_id` be not a new email ? There is not checks that validate such name or `latest_email_id`. You should aim to rename `possible_new_email_id` to just `new_email_id` to simplify logic and names.